taf/updater/handlers.py

```diff
@@ -257,7 +257,15 @@ def cleanup(self):
         self.validation_auth_repo.cleanup()
         temp_dir = Path(self.validation_auth_repo.path, os.pardir).parent
         if temp_dir.is_dir():
-            shutil.rmtree(str(temp_dir), onerror=on_rm_error)
+            try:
+                shutil.rmtree(str(temp_dir), onerror=on_rm_error)
+            except (OSError, PermissionError) as e:
+                taf_logger.error(
```

Comment:
Thinking that this is more of a warning than an error. If there were no other issues, but cleanup failed, that should not result in an error, right? (I know that the broader context is that we have issues when an error occurs)

Reply:
Makes sense, moved message to `on_rm_error` handler as a warning